crates/transaction-emitter-lib/src/emitter/account_minter.rs

```diff
@@ -277,14 +280,28 @@ impl<'t> AccountMinter<'t> {
         amount: u64,
     ) -> Result<()> {
         info!("Minting new coins to root");
+        for i in 0..3 {
+            self.source_account.set_sequence_number(
+                txn_executor
+                    .query_sequence_number(self.source_account.address())
+                    .await?,
+            );
 
```

Comment:
A shared DB among instances or some shared state between threads and emitters should keep track of the sequence number.  Right now you must have multiple items fighting over the same transactions.

The transactions should probably also be queued

Reply:
these runs from different machines (and only once per run)